x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -239,10 +239,8 @@ const TimelineDescriptionComponent: React.FC<FlyoutHeaderProps> = ({ timelineId
   );
   return (
     <EuiText size="s" data-test-subj="timeline-description">
-      {description.length ? (
-        <LineClamp key={description.length} lineClampHeight={4.5}>
-          {description}
-        </LineClamp>
+      {description ? (
+        <LineClamp lineClampHeight={4.5}>{description}</LineClamp>
```

Comment:
Know why `key={description.length}` was defined before?

Reply:
The `LineClamp` is the only child inside the `EuiText` parent, it seems not needed to me as this is not a list.
Maybe there's another reason I am missing. It was introduced by @machadoum in this commit: https://github.com/elastic/kibana/commit/fbbbcf8b9b856c6f7e602973552a4a0c98938372#diff-4f12a0dc265bf0cac080fcea371df61e4806f9b2bfd530c1fb56221e797e3fb0

Reply:
Yea, same. I thought I was missing something as well, but it looks like @machadoum approved it, so no worries 👍🏾

Reply:
Oh, I missed that. `key={description.length}` was intentionally there to force react to recreate the component when `description` changes. When `description` changes `LineClamp` doesn't recalculate its size. A better solution would be to refactor `LineClamp` `useEffect` to use `children` as a dependency.

Reply:
Ok thanks @machadoum, then I will test if this solution works, or bring back the `key` otherwise.